### ISSUE TITLE
Implement updates to VAMC Police data page after collab cycle midpoint review

### ISSUE THREAD

**Issue by xiongjaneg** [department-of-veterans-affairs/va.gov-cms#16329]
## Description or Additional Context 
Design updates were made after Design Intent in #15631. This is a ticket to update designs  and content after Midpoint review feedback

## Non-collab cycle updates
Changes noted so far to [the Sketch file](https://sketch.com/s/b48c905c-7c7f-46fb-9ae5-6020ff0e5174).

Changes made:

- Added Use of force and Weapons discharge definitions
- Split definitions up into 3 additional info components that correspond with each data section.
- Moved How to contact us section above the How to request a police report section

Changes left to make:

- More FAQs will be added.

## Collab cycle midpoint feedback

<details>
  <summary>Design</summary>

Design feedback: https://github.com/department-of-veterans-affairs/va.gov-team/issues/73098

- [x] Must: Per [our guidance](https://design.va.gov/components/additional-info#when-to-consider-something-else), Additional Info components should not have too much content. You should not try to put multiple paragraphs inside Additional info. This includes form fields that require a lot of explanation. Link to another page, consider an [Accordion](https://design.va.gov/components/accordion), or shorten the content.)

</details>

<details>
 <summary>Content</summary>
Content feedback: https://github.com/department-of-veterans-affairs/va.gov-team/issues/73205

- [x] VA Police - This term is inconsistently capitalized throughout the flow. The preferred capitalization is “VA police.” Please revise to ensure that capitalization for this term is consistent throughout the page.

- [x] VA policies - This term is inconsistently capitalized throughout the flow. Please revise to ensure that capitalization for this term is consistent throughout the page.

- [x] VA Police Officers - After checking with CAIA, the preferred capitalization for this term is “VA police officers.” Please revise.

- [x] Current > Tickets > Arrests - The user finds the following sentences: “The number of times that VA Police take custody of a person to face criminal charges. The arrest can be conducted on an existing warrant, or for crimes committed in the officers presence.” There are multiple small errors in this section. Please revise to the following: “The number of times that VA police have taken custody of a person to face criminal charges. The arrest could have been conducted on an existing warrant or for crimes committed in the officers' presence.”

- [x] Current > Tickets > Prosecutions - The user finds the following sentence: “Data about actions taken by VA police following traffic violations or criminal activity.” This explanation is not consistent with the formatting of the other definitions in this section. Is this a number count? Or is it something else? Please revise so this definition is consistent with the others or clearly explains what this data point is/means.

- [x] Current > Tickets > Tickets - The user finds an explanation for what the “Tickets” data point means. They find this last sentence: “Tickets can be issued for violations and can result in fines, or up to 6 months confinement.” There is a small comma error here. Please revise by removing the comma in this sentence.

- [x] Current > Police conduct > Professional standards - The user finds the following explanation for professional standards: “Data about VA police conduct.” This explanation is not consistent with the formatting of the other definitions in this section. Is this a number count? Please revise so this definition is consistent with the others or clearly explains what this data point is/means.

- [x] Current > Use of force (nationwide) > Use of Force - The term “use of force” is inconsistently capitalized throughout the section. Please revise to ensure that capitalization for this term is consistent throughout the section and the page.

- [x] Current > Use of force (nationwide) > Weapons Discharge - Under “What do these terms mean?”, the user finds a definition for “Weapons Discharge.” “Discharge” shouldn’t be capitalized in this case. Please revise.

- [x] Unhappy Paths > Contact information - The user finds the following sentence: “Contact us to request information regarding the arrest, ticketing, detainment, use of force, or other police matters.” This is awkwardly worded due to the “the” before the list. Please revise so the sentence reads as: “...information regarding arrests, ticketing detainment, use of force, or other police matters.” _Note: The Unhappy paths page isn't up to date. This sentence will be removed once the page is updated._

</details>

<details>
<summary>Accessibility</summary>
Accessibility feedback: https://github.com/department-of-veterans-affairs/va.gov-team/issues/73271

- [x] Must: When choosing the police activity information to display, add a button after the three dropdowns that will trigger the action to display data, rather than having the action occur just from making a selection with the dropdowns. My expectation would be that focus moves when triggering the action (more on that below), and so triggering that focus change via a button avoids situations where a user is arrowing down through dropdown options and having their focus move unexpectedly before they've made their selections.

- [x] Laura will annotate in Sketch
     - Coding: When selecting the data to display, keyboard focus should move to the "[month and year] police activity at [system name]" heading. This ensures that screen reader users know something has happened --- by moving focus, that heading gets immediately announced to tell them what's now displayed, and the data they're looking for is the next thing they'll encounter. 
     - Likewise for the unhappy paths, keyboard focus should move to the first heading for the information that's generated. I don't think all of the unhappy paths include a message indicating what is/isn't available --- would be great to have something consistent across all the scenarios. But regardless, keyboard focus should move to the first relevant heading.

</details>

## Acceptance Criteria
- [x] Use of force and Weapons discharge definitions are added.
- [x] Definitions are split into 3 additional info components that correspond with each data section.
- [x] How to contact us section is moved above How to request a police report section.
- [x] ~~FAQs (AC update pending)~~ _Note: I'm not sure this is relevant anymore._
- [x] Collab cycle design MUSTs are addressed (note in comments)
- [x] Collab cycle content MUSTs are addressed (note in comments)
- [x] Collab cycle accessibility MUSTs are addressed (note in comments)
- [x] Requires accessibility review
- [x] Request UX Lead review (usually 48 hours turnaround)
- [ ] Tag PM to stub subsequent engineering tickets

**Comment by davidmpickett** [comment]
This should wait until after collab cycle Midpoint Review, since that may have additional design changes

**Comment by xiongjaneg** [comment]
@thejordanwood @davidmpickett Would you please refine/point this for next sprint when you get a chance? Thank you!

**Comment by davidmpickett** [comment]
I agree with Jordan's 5. Most of this from a content perspective will just be confirming that the feedback is out of sync with the content source of truth

**Comment by davidmpickett** [comment]
I confirmed that all the [Content Feedback](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73205) is addressed in the [content source of truth](https://dvagov.sharepoint.com/:w:/s/SitewideCAIA/ESKYtD-cnh9Pg8rBf_kgYaoBmr8Mg7eMTrtKtKoGrXCDiA?e=IQyeOD)

**Comment by davidmpickett** [comment]
I commented on the [Design feedback](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73098) with updated copy to see if we are good to use the Additional info component.

**Comment by thejordanwood** [comment]
Thanks @davidmpickett! I've updated [the Sketch file](https://sketch.com/s/b48c905c-7c7f-46fb-9ae5-6020ff0e5174) to show the updated copy in the additional info components and commented that on the Design feedback ticket. Hopefully that helps!

**Comment by thejordanwood** [comment]
@laflannery Thanks for leaving comments in the Sketch file. I've converted them to our usual annotations. I wanted to capture them before I move this file to Figma next sprint because all comments will be lost after that. 

Some other notes:

- I have a question in the Sketch file about what to label the "Submit" button. Alternatives could be "Search" or "Filter." Or maybe "Submit" is okay and I'm overthinking it.
- I updated the Unhappy paths page and was able to condense some scenarios. Other scenarios were removed because they focused on what happens if there is no contact info, which is no longer relevant. One comment still lingering is what to do when a month or year has no data. I put a suggestion on that comment thread, but we may need to ask the team.

**Comment by laflannery** [comment]
@thejordanwood I don't mind submit at all however I am leaning towards "Search" because on the VA site that seems to be what we use for filter type searching (We use "Search" on the Location filters) so keeping this consistent would be good.

**Comment by thejordanwood** [comment]
@laflannery The button now says "Search"

**Comment by davidmpickett** [comment]
This wasn't technically an AC, but I also just updated the [content source of truth](https://dvagov.sharepoint.com/:w:/s/SitewideCAIA/ESKYtD-cnh9Pg8rBf_kgYaoBmr8Mg7eMTrtKtKoGrXCDiA?e=IQyeOD) to reflect additional feedback from CAIA that came in with midpoint. There are still a couple small lingering questions, but should be pretty easy to wrap up once we actually get data and start building.

**Comment by davidmpickett** [comment]
@aklausmeier Ready for review!

**Comment by aklausmeier** [comment]
@davidmpickett and @thejordanwood As I was reviewing Sketch against Midpoint feedback and content doc I noticed new content updates from Michelle. Will you ping me for review after these minor updates have been made in the Sketch file?

**Comment by thejordanwood** [comment]
@aklausmeier We've addressed the new content updates from Michelle. This should be ready for you to review now.

**Comment by aklausmeier** [comment]
@thejordanwood reviewed, looks great!

**Comment by thejordanwood** [comment]
@xiongjaneg Reviews are complete on this ticket. This can close whenever you're ready.
